bitnami/jaeger/templates/query/deployment.yaml

```diff
@@ -0,0 +1,201 @@
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+  replicas: {{ .Values.query.replicaCount }}
+  {{- if .Values.query.updateStrategy }}
+  strategy: {{- toYaml .Values.query.updateStrategy | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: query
+        {{- if .Values.query.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.query.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+      {{- if .Values.query.podAnnotations }}
+      annotations:
+        {{- if .Values.query.podAnnotations }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.query.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+      {{- end }}
+    spec:
+      {{- if .Values.query.schedulerName }}
+      schedulerName: {{ .Values.query.schedulerName }}
+      {{- end }}
+      priorityClassName: {{ .Values.query.priorityClassName | quote }}
+      {{- if .Values.query.affinity }}
+      affinity: {{- include "common.tplvalues.render" (dict "value" .Values.query.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.query.podAffinityPreset "component" "query" "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.query.podAntiAffinityPreset "component" "query" "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.query.nodeAffinityPreset.type "key" .Values.query.nodeAffinityPreset.key "values" .Values.query.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.query.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.query.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.query.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.query.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.query.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.query.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.query.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.query.topologySpreadConstraints "context" $) | nindent 8 }}
+      {{- end }}
+      initContainers:
+        - name: jaeger-cassandra-ready-check
+          image: {{ printf "%s/%s:%s" .Values.cassandra.image.registry .Values.cassandra.image.repository .Values.cassandra.image.tag }}
```

Comment:
I would expect to use the helpers we have in the common library. Example for Solr.

```
{{/*
Return the proper Apache Solr image name
*/}}
{{- define "solr.image" -}}
{{- include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) -}}
{{- end -}}

```

Reply:
Done [4c35857](https://github.com/bitnami/charts/pull/13480/commits/4c35857292e1531416284ad520b7020e606fd81d)